features/evaluations/eval-types.mdx

```diff
@@ -30,6 +30,10 @@ The payload will be in the form of
         }
     ```
 
+### MCP
+
+The _MCP Action_ evaluation allows you to run functions on a remote MCP server. Simply plug in your server URL and auth, select your function and you will be able to call your function with inputs mapped from other cells.
```

Comment:
Don't call it an "evaluation"

And can you link to MCP docs

I think we can also add a question in the FAQ "Does PromptLayer support MCP?" and put a little screen recording of an agent there?